packages/core/src/__tests__/unit/application-lifecycle.unit.ts

```diff
@@ -13,6 +19,7 @@ import {
   LifeCycleObserver,
   Server,
 } from '../..';
+import {lifeCycleObserver} from '../../lifecycle';
```

Comment:
Please import from `../..`, just add the new imported symbol to the existing import above.